b2share/modules/records/cli.py

```diff
@@ -54,6 +55,14 @@ def check_dois(update):
         check_record_doi(record, update)
 
 
+@b2records.command()
+@with_appcontext
+def update_expired_embargoes_now():
```

Comment:
True now in this sense doens't fit, this was to avoid declaring the same function name. Changed back.